packages/augur-ui/src/modules/market/containers/trading-form.ts

```diff
@@ -63,6 +64,7 @@ const mapStateToProps = (state, ownProps) => {
     availableEth: createBigNumber(state.loginAccount.balances.eth),
     availableDai: createBigNumber(state.loginAccount.balances.dai),
     hasFunds,
+    currentTimestamp: getValue(state, "blockchain.currentAugurTimestamp"),
```

Comment:
I'd like to get away from `getValue` we don't get type checking with it. or convert `getValue` to be a generic method